pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -258,3 +265,15 @@ func parseJSONValue(value string) (map[string]string, error) {
 
 	return result, nil
 }
+
+// getPersistentVolumeClaimName gets the persistent volume names from a statefulset pod
+// returns empty slice if no persistent volume claim was found
+func getPersistentVolumeClaimName(pod *kubelet.Pod) []string {
```

Comment:
```suggestion
func getPersistentVolumeClaimNames(pod *kubelet.Pod) []string {
```

Reply:
maybe we should have this directly on `func (p *Pod) getPersistentVolumeClaimNames()` so that we can benefit from it every where in the code as we don't require any other parameter?

Reply:
yes, makes sense